lua/telescope/builtin/init.lua

```diff
@@ -322,4 +352,46 @@ builtin.lsp_document_diagnostics = require('telescope.builtin.lsp').diagnostics
 ---@field hide_filename boolean: if true, hides the name of the file in the current picker (default is false)
 builtin.lsp_workspace_diagnostics = require('telescope.builtin.lsp').workspace_diagnostics
 
+local apply_config = function(mod)
+  local pickers_conf = require('telescope.config').pickers
+  for k, v in pairs(mod) do
+    local pconf = vim.deepcopy(pickers_conf[k] or {})
+    if pconf.theme then
+      local theme = pconf.theme
+      pconf.theme = nil
+      pconf = require("telescope.themes")["get_" .. theme](pconf)
+    end
+    if pconf.mappings then
+      local mappings = pconf.mappings
+      pconf.mappings = nil
+      pconf.attach_mappings = function(_, map)
+        for mode, tbl in pairs(mappings) do
+          for key, action in pairs(tbl) do
+            map(mode, key, action)
+          end
+        end
+        return true
+      end
+    end
+    mod[k] = function(opts)
+      opts = opts or {}
+      if pconf.attach_mappings and opts.attach_mappings then
+        local attach_mappings = pconf.attach_mappings
+        pconf.attach_mappings = nil
+        local opts_attach = opts.attach_mappings
```

Comment:
is it possible for this to be `nil`?

Reply:
Oh, I should just read the code above LUL

Reply:
There is an if check above but i can make it more explicit with `~= nil` because true would also pass here